cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporter.java

```diff
@@ -162,20 +165,48 @@ private void createCruiseControlMetricsTopic() {
       createTopicsResult.values().get(_newTopic.name()).get();
       LOG.info("Cruise Control metrics topic created: {}", _cruiseControlMetricsTopic);
     } catch (InterruptedException | ExecutionException e) {
-      if (!(e.getCause() instanceof TopicExistsException)) {
-        LOG.error("Unable to create Cruise Control topic", e);
-      }
+      LOG.warn("Unable to create Cruise Control metrics topic " + _cruiseControlMetricsTopic, e);
```

Comment:
Nit: Should we use lazy logging to avoid string concat? -- i.e.
`LOG.warn("Unable to create Cruise Control metrics topic {}.", _cruiseControlMetricsTopic, e);`